frontend-cert/js-projects/rock-paper-scissor/user-stories.md

```diff
@@ -0,0 +1,10 @@
+1. Declare a variable `year` that stores the value of the year to check.
+
+1. Use an `if/ else` statement to check if the year is a leap year.
+
+1. To check if the year is a leap year, fulfill the following conditions in the `if` part of the statement:
+- If the year is divisible by 4, then it is a leap year.
+- If the year is divisible by 100, then it is not a leap year.
+- If the year is divisible by 400, then it is a leap year.
+
+1. If the year is a leap year, print `year + " is a leap year."` to the console. Otherwise, print `year + " is not a leap year."`.
```

Comment:
I might reword this another way.
Right now it makes this use of syntax makes it seem like they have to use this type of string concatenation and can't use template literals.

Maybe something like this

```suggestion
1. If the year is a leap year, output `[year] is a leap year.` to the console. Otherwise, output `[year] is not a leap year.`. You will replace [year] with the actual year.
```